ParallaxHeader.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "ParallaxHeader"
-  s.version      = "2.0.0"
+  s.version      = "2.0.1"
```

Comment:
I believe it should a major upgrade, since it has breaking changes

Reply:
We could also specify the `s.swift_version = '4.2'` for the Pod to be built on the correct version